### ISSUE TITLE
dialyzer stops building plt due to embedded yaws dependency.

### ISSUE THREAD

**Issue by iambumblehead** [klacke/yaws#194]
dialyzer stops building plt due to embedded yaws dependency.

I'm unsure if this a problem with yaws or my use of dialyzer. I'm running dialyzer from the root directory of my application with this:
```
$ dialyzer --add_to_plt --apps ssl kernel stdlib erts mnesia eunit -r deps
dialyzer: Could not get abstract code for file: /home/duko/Software/gani/deps/yaws/ebin/advanced_echo_callback.beam (please recompile it with +debug_info)
```
I modified yaws' rebar.config to use `debug_info` rather than `no_debug_info` and used rebar compile but this did not resolve my issue.

**Comment by vinoski** [comment]
Did you delete the `deps/yaws/ebin/*.beam` files after making the `rebar.config` change, to force them to rebuild?

I'm going to change the yaws `rebar.config` to remove the `no_debug_info` setting.

**Comment by vinoski** [comment]
I removed the `no_debug_info` setting on yaws master.

**Comment by iambumblehead** [comment]
vinoski,

I have just created a plt file with using the yaws master. Thank you.

**Comment by vinoski** [comment]
Great! In that case I'll close this this issue as fixed in 5d755a6.
